src/parse.js

```diff
@@ -12,32 +12,44 @@ module.exports = function(text) {
   if (!datesMatch) { throw new Error('Failed to parse issued dates'); }
 
   // Capture the data rows
-  let dataMatch = text.match(/(\d{2})-(\d{2})UT\s+(\d)\s+(\d)\s+(\d)/g);
+  let dataMatch = text.match(/(\d{2})-(\d{2})UT\s+(\d.\d{2}).*(\d.\d{2}).*(\d.\d{2})/g);
   if (!dataMatch || !dataMatch.length == 8) { throw new Error('Failed to find data'); }
 
   // Parse the data rows
   let breakdown = [];
   for (let i = 0; i < dataMatch.length; i++) {
-    let lineMatch = dataMatch[i].match(/(\d{2})-(\d{2})UT\s+(\d)\s+(\d)\s+(\d)/);
+    let lineMatch = dataMatch[i].match(/(\d{2})-(\d{2})UT\s+(\d.\d{2}).*(\d.\d{2}).*(\d.\d{2})/);
     if (!lineMatch) { throw new Error('Failed to parse line'); }
 
     // Store entry for each column
     breakdown.push({
       start: new Date([year, datesMatch[1], datesMatch[2], lineMatch[1] + ':00', 'UTC'].join(' ')),
-      kp: lineMatch[3]
+      kp: Math.round(lineMatch[3]).toString()
     });
     breakdown.push({
       start: new Date([year, datesMatch[3], datesMatch[4], lineMatch[1] + ':00', 'UTC'].join(' ')),
-      kp: lineMatch[4]
+      kp: Math.round(lineMatch[4]).toString()
     });
     breakdown.push({
-      start: new Date([year, datesMatch[5], datesMatch[6], lineMatch[1] + ':00', 'UTC'].join(' ')),
-      kp: lineMatch[5]
+      start: new Date([year, datesMatch[5], datesMatch[6],lineMatch[1] + ':00', 'UTC'].join(' ')),
+      kp: Math.round(lineMatch[5]).toString()
     });
   }
-
+  breakdown = breakdown.sort(GetSortOrder());
```

Comment:
[Array.prototype.sort](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort) works in-place, so the assignment isn't necessary here.